README.md

```diff
@@ -6,16 +6,57 @@ This module focuses on the analysis of RNA-seq data.
 Here is the [schedule](schedule.md) for the workshop.
 
-## Post-Docker Steps
-
-1. In Kitematic, search for "CCDL". Get the training_rnaseq image.
-2. Start the training_rnaseq image.
-3. Click the settings button.
-4. On the network panel you should see that an IP address has been assigned in the PUBLISHED IP:PORT column. 
-![Published IP](screenshots/all-01-network.png)
-5. On the volume panel you should see that a folder has been assigned. This is where output files can be stored so that they will show up on your computer. 
+## Post-Docker Steps using Kitematic
+
+1. Pull the appropriate image using command line.
+
+- In *Mac*, search for `Terminal` and open that application.
+- In *Windows*, go to `Application` > `Utilities` > `Command Prompt`
+
+  In your respective command line interface, copy and paste the following:
+```
+docker pull ccdl/training_rnaseq:2019-houston
+```
+
+2. Run the container. Change the PASSWORD in the line below to whatever you'd
+  like.
+```
+docker run -e PASSWORD=<PASSWORD> -p 8787:8787 ccdl/training_rnaseq:2019-houston
+```
+
+3. Open `Kitematic` - you should see an image running.
+
+
+4. `Settings` > `Volumes` > Set local folder to "main directory" that was
+transferred from the flash/hard drive, using the `CHANGE` button.
 ![Folder](screenshots/all-02-volume.png)
-6. On the advanced panel you should that a TTY is assigned and STDIN will remain open. 
-![Advanced](screenshots/all-03-advanced.png)
-7. Visit the IP address that was listed in the PUBLISHED IP tab in a web browser and you should see an RStudio login. The login information is username: `rstudio` and password: `rstudio`. If you do not see this, raise your hand. 
-![RStudio](screenshots/all-04-rstudio.png)
+
+
+5. Navigate to RStudio window.
+
+  - In a *Mac*, you can navigate to the RStudio window by typing
+  `localhost:8787` in your web browser.
+
+  - In a *Windows*, go to the `Home` tab and look for the `ACCESS URL` in
+  the `IP & PORTS` pane. Copy this URL and paste it into the navigation bar of
+  your web browser.
+
+
+6. Log into `RStudio`. The username will be 'rstudio' and the password will be
+whatever you selected above (can also be accessed from the `Settings` >
+`General` panel).
+
+
+7. You should see a kitematic/ folder in your `RStudio` Files panel. Click on it.
+If you do not see the training modules folders in the kitematic folder, raise
+your hand.
+
+
+### If Kitematic doesn't work:
+If all else fails and Kitematic is not working for you, go to your Terminal or
+Command Prompt (for Mac or Windows respectively) and type in the following, but
+replacing <PATH_TO_TRAINING_FOLDERS> bit with the absolute path to
+"main directory" that was transferred from the flash/hard drive.
+```
+docker run -it --rm --mount type=volume,dst=/home/rstudio/kitematic,volume-driver=local,volume-opt=type=none,volume-opt=o=bind,volume-opt=device=<PATH_TO_TRAINING_FOLDERS> -e PASSWORD=<PASSWORD> -p 8787:8787
+```
```

Comment:
This doesn't cover the getting to RStudio in your browser step.

Reply:
Oh. I guess I should just put that they can follow the other steps the same way. (Although I'm just presuming that with the Windows).